### ISSUE TITLE
Android 9, works properly but fails when TalkBack is active

### ISSUE THREAD

**Issue by wilsolutions** [skwasjer/skwas-cordova-plugin-datetimepicker#29]
So, as I mentioned, the plugin works fine, the only problem that I have is that when using Android 9 with TalkBack active it simply does not open the component, once I disable TalkBack it works fine.

I have no errors in chrome inspector, I didn't check native Android logs.

**Comment by wilsolutions** [comment]
closing, not an issue, I was tapping in the component instead of swiping to it...
